### ISSUE TITLE
Add a feature in `Repeat tab` to send a request in parallel(To test for race condition)

### ISSUE THREAD

**Issue by biniam-bin** [caido/caido#814]
Its nice to have a feature that sends 10 or 20 requests at the same time. so that testers can test for race conditions

**Comment by sw33tLie** [comment]
This will unfortunately never be as good as it can get until we have HTTP/2 support for the single-packet ones

**Comment by Corb3nik** [comment]
Gonna close this as a duplicate of https://github.com/caido/caido/issues/705

Thanks for reporting!
